src/lib/libsdl.js

```diff
@@ -3122,7 +3122,11 @@ var LibrarySDL = {
       // valid. Check that explicitly and fall back to a plain Canvas if we need
       // to. See https://github.com/emscripten-core/emscripten/issues/16242
       if (typeof SDL.ttfContext.measureText != 'function') {
-        abort('bad context');
+        // According to https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvasRenderingContext2D
+        // OffscreenCanvasRenderingContext2D.measureText() appeared in
+        // Chrome 69, Firefox 105 and Safari 16.4. Fall back to using regular
+        // Canvas2D if OffscreenCanvas2D exists, but does not look workable.
```

Comment:
Is this comment just adding more detail the comment above?  If so, maybe merge them into a single comment?

Ideally we could wrap this whole block in some `MIN_XX_VERSION` checks, so that we clean it up one day, but that doesn't need to part of this PR I guess.

Reply:
These version numbers (Chrome 69, Firefox 105 and Safari 16.4) looks like the same as the ones that support  `OffscreenCanvas` at all, but the comment seems to relate to `measureText`.   ?

Reply:
Removed the old comment.

> These version numbers (Chrome 69, Firefox 105 and Safari 16.4) looks like the same as the ones that support `OffscreenCanvas` at all

Maybe those versions are aggregations of "these versions have the full OffscreenCanvas spec." Firefox 91 ESR definitely does have `new OffscreenCanvas`, but no `.measureText()` here.